src/ir/cost.h

```diff
@@ -546,7 +546,7 @@ struct CostAnalyzer : public OverriddenVisitor<CostAnalyzer, Index> {
   }
   Index visitTry(Try* curr) {
     // We assume no exception will be thrown in most cases
-    return visit(curr->body) + maybeVisit(curr->catchBody);
```

Comment:
Not sure why we were adding `catchBody` in costs, while the comment says we assume no exception will be thrown. Fixed to only count the try body.